shared/modules/macros.jinja

```diff
@@ -88,3 +88,80 @@ ocil: |
 
 {{{ audit_syscall_check_ocil_clause_entry() }}}
 {{%- endmacro %}}
+
+
+{{%- macro _firewalld_check(access_action, port, proto, service) %}}
+    To determine if <code>firewalld</code> is configured to {{{ access_action }}} to <code>{{{ service }}}</code>
+    on port <code>{{{ port }}}/{{{ proto }}}</code>, run the following command(s):
+    {{% if port %}}
+        <code>firewall-cmd --list-ports</code>
+    {{% endif %}}
+    {{% if service %}}
+        <code>firewall-cmd --list-services</code>
+    {{% endif %}}
+{{%- endmacro %}}
+
+
+{{%- macro firewalld_allow_access(port, proto, service) %}}
+{{{ _firewalld_check("allow access", port, proto, service) }}}
+    If <code>firewalld</code> is configured to allow access through the firewall, something similar to the following will be output:
+    {{% if service %}}
+    If it is a service:
+    <code>&gt;{{{ service }}}</code>
```

Comment:
Why is this > here? I know it was there before, but maybe it should not be there.

Reply:
I got the impression that it symbolizes beginning of output of the command. I will be happy to take it out though.